perception/lidar_centerpoint_tvm/lib/network/scatter.cpp

```diff
@@ -62,7 +62,7 @@ void scatterFeatures_worker(
   }
 }
 
-void scatterFeatures(
+[[maybe_unused]] void scatterFeatures(
```

Comment:
I think we want to keep this function as an example to rewrite cuda kernels using cpp, even though it is not being used anymore.

Reply:
I generally don't like to have unused code laying around. It gets hard to remember either what it did or how it was called and it can become hard to interface with the rest as code evolves. What about removing the function but putting a comment, either in the documentation or somewhere relevant in the codebase, pointing to the commit hash that was using it and saying it is a way to to the implementation in CPP?